src/node_interface.rs

```diff
@@ -34,6 +34,7 @@ pub enum MethodCall {
     Post,
     /// request to refresh
     Refresh { content: Box<Sendable>, },
+    RefreshNew { type_tag: u64, from_group: NameType, payload: Vec<u8>, },
```

Comment:
Go ahead and already rename the MethodCall::Refresh with the new signature; it ensures that we have a unique behaviour for this Refresh call